pkgs/development/haskell-modules/patches/cachix-cnix.patch

```diff
@@ -0,0 +1,438 @@
+diff --git a/cachix.cabal b/cachix.cabal
+index 59f94f1..1253e11 100644
+--- a/cachix.cabal
++++ b/cachix.cabal
+@@ -40,7 +40,7 @@ source-repository head
+   location: https://github.com/cachix/cachix
+ 
+ library
+-  import:            defaults
++  import:          defaults
+   exposed-modules:
+     Cachix.Client
+     Cachix.Client.Commands
+@@ -59,41 +59,38 @@ library
+     Cachix.Client.Retry
+     Cachix.Client.Secrets
+     Cachix.Client.Servant
+-    Cachix.Client.Store
+-    Cachix.Client.Store.Context
+     Cachix.Client.URI
+     Cachix.Client.WatchStore
+     System.Nix.Base32
+ 
+-  hs-source-dirs:    src
+-  other-modules:     Paths_cachix
```

Comment:
Due to formatting this patch is bigger than needed.

Reply:
The patch is taken directly from the pull request. I wouldn't have formatted if it wasn't required by their CI. I'd prefer to keep the patch close to the pr.